src/main/java/games/strategy/engine/lobby/server/userDB/DerbyUserController.java

```diff
@@ -156,18 +165,20 @@ public boolean login(final String userName, final String hashedPassword) {
   }
 
   @Override
-  public DbUser getUser(final String userName) {
+  public DbUser getUserByName(final String userName) {
```

Comment:
There are a lot of different variations of "username" (UserName, Name, username). It seems unclear whether it should be referred to as "user name" vs "username" vs "name".